### PR TITLE
Add the cargo assistant job and spawner

### PR DESCRIPTION
**Pull request by Radezolid**
## About the PR
This role is meant to be a learner role for logistics. It spawns with a Logistics 101 guidebook in their pocket like other learning roles. This role makes the cargo tech the non-entry level role now and so I put time requirements on it.

Note: This won't add the job yet nor it limits the cargo tech time. Once it's mapped on every map we can just toggle it.

## Why / Balance
This is meant to act as a easy way to recognize newbies from regular tecchies, like TA with an Engie. It also acts as a barrier from new players from being able to order stuff + it doesn't get salvage access on skeleton crew (Although I doubt this will stop any raiders with a toolbox).

## Technical details
YAML, locale and some sprites changes.

## Media
image
image
image
image
image

## Requirements

- [X] I have tested all added content and changes.
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

Hopefully nothing.

**Changelog**

No CL.

### CODE REVIEW

**Comment by Lyndomen** [comment]
Requesting a hold on merging this, even when it's completely done. We're strategizing a small logistics change, and don't want to have our mappers duplicate work in adding spawners and stuff. 

This is certainly great, and 100% where we want to go in the future, this is just one of a few changes that should go sequentially

**Comment by Lyndomen** [comment]
Under Direction review, 22 hours

**Comment by Radezolid** [comment]
> Under Direction review, 22 hours

Hell yeah, preferential treatment, two hours less than usual.

File: Resources/Prototypes/DeltaV/Loadouts/loadout_groups.yml
Comment (Lyndomen) [inline Resources/Prototypes/DeltaV/Loadouts/loadout_groups.yml:1]: Whitespace :agony:

**Review by Lyndomen — DISMISSED** [review]

**Comment by Lyndomen** [comment]
I'm thinking the go live of this will be to merge this, get maps updated and update playtime recs on the same day

**Comment by Radezolid** [comment]
> I'm thinking the go live of this will be to merge this, get maps updated and update playtime recs on the same day

That might require either coordination of the whole maps mantainers or getting permission from them and just doing it in one PR.

**Comment by Lyndomen** [comment]
> > I'm thinking the go live of this will be to merge this, get maps updated and update playtime recs on the same day
> 
> That might require either coordination of the whole maps mantainers or getting permission from them and just doing it in one PR.

I'm not really sure how else to do it tbh, but yeah it's gonna be a pain

**Review by deltanedas — DISMISSED** [review]

**Review by Colin-Tel — APPROVED** [review]
cool, awesome, hoo-ray!!

**Pull request merged**